### ISSUE TITLE
Some buggy icons in Linux Mint

### ISSUE THREAD

**Issue by felipe-heredia** [snwh/paper-icon-theme#738]
A few days ago, Paper-icons were updated, and on my Linux Mint, some icons were bugged.

[See the image](http://i.imgur.com/d7W1mwL.png).

I ended up not digging through the system to find out if there are any more buggy icons.
I hope it will be fixed soon.

Sorry if the message was not correct, I did it by Google translator.

**Comment by nicosensei** [comment]
I experience the same issue on Linux Mint 19.3 Cinnamon.

**Comment by esquinas** [comment]
I am using Linux Mint and this was an issue from yesterday. Today I updated paper-icon-theme and everthing looks fine at the moment.

To be specific, the version that broke icons on Linux Mint for me was:
paper-icon-theme 1.5.726-202003111925~daily~ubuntu18.04.1

And today this version fixed it:
`paper-icon-theme 1.5.728-202003121505~daily~ubuntu18.04.1`

Older version that also worked fine was:
`paper-icon-theme 1.5.723-201905252133~daily~ubuntu18.04.1`

So if you have the `1.5.723-...` version or earlier, don't update to the `1.5.726-...`, update directly to the `1.5.728-...` version.

**Comment by esquinas** [comment]
By the way, because of that issue, yesterday I used Mint-Y icons, which were the best ones already installed that I could find. And after comming back to Snwh's Paper Icons, I realised I  needed to speak out about how great paper-icon-theme are and how much more praise and attention these icons deserve, really.

I am even thinking about opening a new issue asking @snwh to make a little styleguide so more people, (less skilled ones at designing, like me) can participate, contribute and make new icons. A good example of such styleguide would be [the OpenMoji Styleguide here](https://openmoji.org/styleguide/).

Thanks again for these awesome icons!
